modules/ROOT/pages/debugging-the-runtime-manager-agent.adoc

```diff
@@ -1,15 +1,84 @@
+image::rtm-agent-version.png[Runtime Manager agent version on the server Dashboard page]
+.. Check the xref:release-notes::runtime-manager-agent/runtime-manager-agent-release-notes.adoc[Runtime Manager Agent Release Notes] to determine the latest agent version compatible with your Mule version.
+.. Update the agent, if necessary.
++
+See xref:installing-and-configuring-runtime-manager-agent.adoc#update-the-agent[Update the Agent].
+. Verify that the license and certificate are not expired.
++
+.. In the Runtime Manager navigation menu, select *Servers* and then click the server name.
+The expiration dates appear on the *Dashboard* page:
++
+.The arrow shows the expiration dates on the server *Dashboard* page.
+image::rtm-server-expirations.png[Expiration dates on the server Dashboard page]
++
+//DASHBOARD SHARED 
+include::partial$cluster-server.adoc[tag=viewServerinClusterOrGroup]
++
+* If the license has expired, see xref:mule-runtime::installing-an-enterprise-license.adoc[Install an Enterprise License].
+* If the certificate has expired, see xref:servers-cert-renewal.adoc[Renew Server Certificates].
+. Verify connectivity with all required hosts.
+.. Ensure that your allowlist includes the required hostnames.
++
+See xref:rtm-agent-whitelists.adoc[Ports, IP Addresses, and Hostnames to Whitelist].
+.. Verify that your firewall or proxy server is not intercepting the Runtime Manager certificate.
++ 
+See xref:rtm-agent-proxy-config,adoc#verify-that-the-proxy-server-does-not-modify-the-runtime-manager-certificate[Verify That the Proxy Server Does Not Modify the Runtime Manager Certificate].
+. Verify that the `mule-agent.yml` file is not corrupted or empty.
++
+If the `<MULE_HOME>/conf/mule-agent.yml` file is empty or contains unknown characters:
++
+
+* If you have a backup, use it to restore the corrupted `mule-agent.yml` file.
+* If you don't have a backup, follow these steps:
++
+.. Remove the Mule server from Runtime Manager.
++
+See xref:servers-actions.adoc[Shut Down, Restart, or Delete Servers].
+.. Uninstall the agent. 
++
+See xref:installing-and-configuring-runtime-manager-agent.adoc#uninstall-the-agent[Uninstall the Agent].
+.. Update the agent to the latest supported version.
++
+See the xref:release-notes::runtime-manager-agent/runtime-manager-agent-release-notes.adoc[Runtime Manager Agent Release Notes] and xref:installing-and-configuring-runtime-manager-agent.adoc#update-the-agent[Update the Agent].
+.. Add the Mule server to Runtime Manager.
++
+See xref:servers-create.adoc[Add Servers to Runtime Manager].
+
+
+== Collect Agent Installation Information for Support 
+
+You can use the `$MULE_HOME/bin/amc_debug` script to collect information used by Support to troubleshoot the Anypoint Runtime Manager agent installation issues.
+If you can't solve problems with the Runtime Manager agent, run the script and save the output before filing a support ticket.
```

Comment:
Please consider something like "If you are not successful in resolving the Runtime Manager agent issues...".

"Can't solve problems" may sound a bit critical of the reader's mental capabilities. 😄

Reply:
Yeah, you're right. I was following Grammarly blindly.